.eslintrc.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
you have lint in the project and yet you pushed codes which has linting errors, and linting was not working as expected